src/Coordination/KeeperDispatcher.cpp

```diff
@@ -442,12 +442,12 @@ void KeeperDispatcher::finishSession(int64_t session_id)
 
 void KeeperDispatcher::addErrorResponses(const KeeperStorage::RequestsForSessions & requests_for_sessions, Coordination::Error error)
 {
-    for (const auto & [session_id, time, request] : requests_for_sessions)
+    for (const auto & [session_id, time, request, zxid, nodes_hash] : requests_for_sessions)
     {
         KeeperStorage::ResponsesForSessions responses;
         auto response = request->makeResponse();
         response->xid = request->xid;
-        response->zxid = 0;
+        response->zxid = zxid;
```

Comment:
actually it's the same, because zxid is not set at this point, thought the function is called at multiple places
I'll return it to 0 as it's more explicit